lib/mix/lib/mix/tasks/deps.unlock.ex

```diff
@@ -27,6 +27,14 @@ defmodule Mix.Tasks.Deps.Unlock do
     Mix.Project.get!()
     {opts, apps, _} = OptionParser.parse(args, switches: @switches)
 
+    config = Mix.Project.config()
+
+    Mix.Project.with_deps_lock(config, fn ->
```

Comment:
```suggestion
    Mix.Project.with_deps_lock(fn ->
```

Reply:
I was doing it proactively, so if the config is needed in the future, we don't fetch it twice, but your call!